accounts/src/main/kotlin/pl/edu/pw/ia/accounts/application/model/CreateAccountRequest.kt

```diff
@@ -1,29 +1,32 @@
 package pl.edu.pw.ia.accounts.application.model
 
 import io.swagger.v3.oas.annotations.media.Schema
-import jakarta.validation.constraints.Email
 import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
 import java.util.UUID
 import org.hibernate.validator.constraints.Length
 import pl.edu.pw.ia.shared.domain.command.CreateAccountCommand
 
 data class CreateAccountRequest(
```

Comment:
This Request (and the corresponding endpoint) got modified to match the signature of the keycloak callee